### ISSUE TITLE
CommandButtons are not accessible with detail template

### ISSUE THREAD

**Issue by jivanova** [telerik/kendo-ui-core#5028]
### Enhancement

Requested in Ticket with ID: 1406932 
[DOJO](https://dojo.telerik.com/IwuQEVEy)

### Reproduction of the problem

Instructions in the Dojo

### Current behavior

Detail Template is toggled on Enter. It is not possible to action the buttons.

### Expected/desired behavior

When the target is any of the Command Buttons, the Grid should focus the buttons/ the template and execute and related actions with the template. 

- if the current cell has expand icon, expand and collapse on enter
- if the current cell has buttons, focus the first one and on the next Enter press, call its function handler

### Workaround

1. Add `editable: "inline"` to the grid configuration, this will allow you to focus the custom command buttons.

[Dojo](https://dojo.telerik.com/IYOqotOB)

2. Add a keydown event on the grid custom command buttons, programmatically focus them on the [navigate](https://docs.telerik.com/kendo-ui/api/javascript/ui/grid/events/navigate) event and stop the immediate propagation of the enter keydown event. [Dojo.](https://dojo.telerik.com/OWuTOYiR)

```
navigate: function(e){             
  if(this.current().hasClass("k-command-cell")){             
     this.current().find(".k-button").first().focus();              
  }
 }
 
$(".k-button").on("keydown", function(e){
 if(e.key == "Enter"){
  e.stopImmediatePropagation();
 }
});
```

### Environment

* **Kendo UI version:** 2019.1.220
* **Browser:** all
